src/Components/Artwork/Badge.tsx

```diff
@@ -0,0 +1,83 @@
+import { Flex, Sans } from "@artsy/palette"
+import { Badge_artwork } from "__generated__/Badge_artwork.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+interface BadgeProps {
+  artwork: Badge_artwork
+}
+
+class Badge extends React.Component<BadgeProps> {
+  render() {
+    const { artwork } = this.props
+    const { is_biddable, is_acquireable, is_offerable, href, sale } = artwork
+    const includeBidBadge = is_biddable || (sale && sale.is_preview)
+    // E.g.(ENDS IN 59M)
+    const saleTimingHint =
+      sale && sale.display_timely_at ? ` (${sale.display_timely_at})` : ""
+    return (
+      <>
+        <Badges>
+          {includeBidBadge && (
+            <Label>
+              <Sans size="0">Bid{saleTimingHint}</Sans>
+            </Label>
+          )}
+          {is_acquireable && (
+            <Label>
+              <a
+                href={href}
+                style={{ textDecoration: "none", cursor: "pointer" }}
```

Comment:
Do these styles need to be applied inline, or can we use styled components like the other elements are using?

Reply:
Good point, this might even be better off as a `Link` anyway. Will clean it up.